### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by AhemdAboAbadi** [GSG-G10/booky#37]
https://github.com/GSG-G10/booky/blob/b75b9b933519878564d35bc671ee0675f6e050bc/public/index.html#L33
`input` tag should have a label for accessibility purposes and if you don't want the label text to be printed on the screen you can use the attributes like `aria-label`
